dev/tasks/conda-recipes/pyarrow/meta.yaml

```diff
@@ -53,6 +55,9 @@ requirements:
     - six
     - futures  # [py27]
     - arrow-cpp {{ ARROW_VERSION }}
+    # - {{ pin_compatible('arrow-cpp') }}
```

Comment:
@xhochy I had to pin arrow-cpp explicitly to pass the builds - I might miss something for pin_compatible to work properly though.

Reply:
I'm also not really sure how it works....